src/WebJobs.Extensions.DurableTask/Options/HttpOptions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Options
+{
+    /// <summary>
+    /// Used for Durable HTTP functionality.
+    /// </summary>
+    public class HttpOptions
+    {
+        /// <summary>
+        /// Reserved name to know when a TaskActivity should be an HTTP activity.
+        /// </summary>
+        public const string HttpTaskActivityReservedName = "Durable:Http:Async:Activity:Function";
+
+        /// <summary>
+        /// Time between the async http requests.
+        /// </summary>
+        public int DefaultAsyncRequestSleepTime { get; set; } = 5000;
```

Comment:
We should make it clear this value is in Milliseconds in the name, as well as the documentation comments. Also, we should decide whether `5000` is the right default.